scene/resources/ray_shape.cpp

```diff
@@ -41,6 +41,10 @@ Vector<Vector3> RayShape::get_debug_mesh_lines() {
 	return points;
 }
 
+real_t RayShape::get_enclosing_radius() const {
+	return length;
```

Comment:
Not really a radius? `length` would be twice as big. In some places it returns a proper radius, sometimes it doesnt.

Reply:
The ray shape is casted always from the origin of the shape, so a sphere / circle that encloses it must have the same radius of the ray length.

However, the final radius of a shape depends also on it's `Shape` node transformation relative its parent.
So if the Ray length is 1 and it's offsetted to x0, y-3, z0 it's the shape total length is 3.